### ISSUE TITLE
Cannot read property 'match' of undefined

### ISSUE THREAD

**Issue by urbanguacamole** [JohnDeved/zooqle-api-ts#1]
Hey!
First up I'd like to thank you for this project. I'm thinking about adding resilient movies search to [Torrent Paradise](https://torrent-paradise.ml/about.html).

While implementing this, I stumbled upon a problem:

```
zooqle.search('tt0076759').then((res) => {
    console.log(res)
})

zooqle.search('tt0468569').then((res) => {
    console.log(res)
})
```

Both IMDB IDs work in browser, but only the second works in the API. I would expect the first request not to throw an exception.

Thanks again!

**Comment by JohnDeved** [comment]
thanks for letting me know!
i will take a look at this as soon as possible

**Comment by JohnDeved** [comment]
@urbanguacamole this issue has been fixed in version "^1.0.46" now.
please don't hesitate to let me know if you run into any other problems.
cheers
